README.md

```diff
@@ -1,47 +1,56 @@
 # Mantle: Gluing CoreOS together
 
-This repository is a collection of utilities for the CoreOS SDK.
-
-## plume
-
-CoreOS release utility
-
-## plume index
-
-Generate and upload index.html objects to turn a Google Cloud Storage
-bucket into a publicly browsable file tree. Useful if you want something
-like Apache's directory index for your software download repository.
+This repository is a collection of utilities for the Container Linux SDK. Most of the
+tools are for uploading, running, and interacting with Container Linux instances running
+locally or in a cloud.
 
 ## kola
 
-Kola is a framework for testing software integration in CoreOS instances
+Kola is a framework for testing software integration in Container Linux instances
 across multiple platforms. It is primarily designed to operate within
-the CoreOS SDK for testing software that has landed in the OS image.
+the Container Linux SDK for testing software that has landed in the OS image.
 Ideally, all software needed for a test should be included by building
 it into the image from the SDK.
 
 Kola supports running tests on multiple platforms, currently QEMU, GCE,
-and AWS. In the future systemd-nspawn and other platforms may be added.
+AWS, VMware ESX and Packet. In the future systemd-nspawn and other
```

Comment:
We should probably call it out directly as `VMware vSphere` then, the current `kola` platform requires clone which isn't available in the free-ware version of the hypervisor, the unlocked version is `vSphere` (even though it's still an `ESX` hypervisor)